src/main/kotlin/value/VList.kt

```diff
@@ -44,8 +45,8 @@ data class VList(var v: MutableList<Value> = mutableListOf()): Value() {
     private fun propSorted(): VList {
         if (v.isEmpty()) return make(v)
         return make(when (v[0]) {
-            is VInt -> v.sortedBy { (it as VInt).v }
-            is VFloat -> v.sortedBy { (it as VFloat).v }
+            is VInt -> v.sortedBy { (it as? VInt)?.v ?: 0 }
+            is VFloat -> v.sortedBy { (it as? VFloat)?.v ?: 0f }
```

Comment:
Unrelated bugfix for sorting mixed lists -- if you sort a list of mixed value types we do our best and assume you want them sorted by the value semantics of the first element.